schema/one-time-polled-screening.json

```diff
@@ -915,7 +915,7 @@ Note that, as this is a demo check, no credits will actually be deducted from th
                 "type": "object"
             },
             "ProviderData": {
-                "description": "The structured JSON data returned by the data provider, or a representation of the provider's response, to allow PassFort to investigate issues that may arise with your integration.",
+                "description": "The structured JSON data returned by the data provider, or a representation of the provider's response, to allow Passfort to investigate issues that may arise with your integration.",
```

Comment:
Remove both commas

Reply:
Change "...to investigate issues that may arise..." to "...to investigate any issues that arise..."